component/main.jsonnet

```diff
@@ -17,34 +17,30 @@ local admin_secret = kube.Secret(params.admin.secretname) {
   },
 };
 
-local external_db_secret =
-  local isdummysecret =
-    if params.database.builtin then
-      {
-        'commodore.syn.tools/dummy-secret': 'true',
-      }
-    else
-      {};
-  kube.Secret(params.database.external.secretname) {
-    metadata+: {
-      labels+: params.labels + isdummysecret,
-    },
-    stringData:
-      if !params.database.builtin then
-        {
-          DB_VENDOR: params.database.external.vendor,
-          DB_ADDR: params.database.external.host,
-          DB_PORT: params.database.external.port,
-          DB_DATABASE: params.database.external.database,
-          DB_USER: params.database.external.username,
-          DB_PASSWORD: params.database.external.password,
-        }
-      else {},
-  };
+local secrets = {
+  builtin: {
+    'postgresql-password': params.database.password,
```

Comment:
It will inject the env var postgresql-password, which is probably not needed. But I think this is acceptable.

Reply:
correct. I verified locally with KIND and it didn't cause issues resp. it gets ignored.